Formula/fontforge.rb

```diff
@@ -105,6 +106,9 @@ def caveats; <<-EOS.undent
 
     Alternatively, install with Homebrew-Cask:
       brew cask install fontforge
+
+    As a last resort, you may try the *unsupported* option:
+      brew install fontforge --with-x11
```

Comment:
Given we're removed it previously I'd rather not re-add it until we have something that works without X11.